ljmd.py

```diff
@@ -1,5 +1,164 @@
+"""
+Python wrapper for LJMD C code
+
+Author: Krister Jazz Urog
+Date: 17 February, 2020
+"""
 from ctypes import *
+import sys
+
+class MDSYS_T(Structure):
+	"""
+	Molecular dynamics system structure object wrapper for a similar C struct.
+
+	-----------------------------------------------
+	Parameters:
+		natoms = number of atoms
+		nfi = current iteration step
+		nsteps = total number of iteration steps
+		dt = time step
+		mass = mass of atom
+		epsilon = constant
+		sigma =  constant
+		box = box size
+		rcut = cutoff threshold distance
+		ekin = Kinetic energy 
+		epot = Potential energy
+		temp = Temperature
+	"""
+	_fields_ = [('natoms', c_int), ('nfi', c_int), ('nsteps', c_int),
+				('dt', c_double), ('mass', c_double), ('epsilon', c_double),
+				('sigma', c_double), ('box', c_double), ('rcut', c_double),
+				('ekin', c_double), ('epot', c_double), ('temp', c_double), 
+				('rx', POINTER(c_double)), ('ry', POINTER(c_double)), ('rz', POINTER(c_double)), 
+				('vx', POINTER(c_double)), ('vy', POINTER(c_double)), ('vz', POINTER(c_double)), 
+				('fx', POINTER(c_double)), ('fy', POINTER(c_double)), ('fz', POINTER(c_double))] 
+
+class LJMD:
+	def __init__(self, so_file):
+		self._ljmd = CDLL(so_file)
+
+	def initialize_system(self, input_file):
+		data = self.read_input(input_file)
+
+		self.natoms = int(data[0])
+		self.mass = float(data[1])
+		self.epsilon = float(data[2])
+		self.sigma = float(data[3])
+		self.rcut = float(data[4])
+		self.box = float(data[5])
+		self.restfile = data[6]
+		self.trajfile = data[7]
+		self.ergfile = data[8]
+		self.nsteps = int(data[9])
+		self.dt = float(data[10])
+		self.nprint = int(data[11])
+
+		self.sys = MDSYS_T(natoms=self.natoms,
+							mass=self.mass,
```

Comment:
Could it be the indentation be fixed in a more proper way?

Reply:
I am not sure, it doesnt look like that on my screen.

![image](https://user-images.githubusercontent.com/47043702/74685716-99e19780-51cf-11ea-91bb-a3d641fbd493.png)

Reply:
There should be a thing with the editors... I will check it later